README.md

```diff
@@ -207,25 +207,28 @@ once the application starts.
 
 ## Run the application <a name="development-run"></a>
 
-1. [Set up the GitHub App](#github-app)
-2. [Set up the application](#setup)
+1. [Set up the GitHub App](#github-app).
+2. [Set up the command-bot application](#setup)
+
+    Along with creating app, replace *placeholders* in `.env.cjs` with required data.
```

Comment:
This is included in the Setup section, more exactly `setup-environment-variables`

Reply:
removed